pkg/api/dtos/models.go

```diff
@@ -79,3 +79,15 @@ func GetGravatarUrlWithDefault(text string, defaultText string) string {
 
 	return GetGravatarUrl(text)
 }
+
+func IsHiddenUser(userLogin string, signedInUser *models.SignedInUser) bool {
+	if signedInUser.IsGrafanaAdmin || userLogin == signedInUser.Login {
+		return false
+	}
+
+	if _, hidden := setting.HiddenUsers[userLogin]; hidden {
```

Comment:
do not access global settings object, we are trying to get rid of this globals access